org.eclipse.scout.rt.shared/src/main/java/org/eclipse/scout/rt/shared/user/Users.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.shared.user;
+
+import org.eclipse.scout.rt.platform.context.RunContext;
+
+public final class Users {
+
+  private Users() {
+  }
+
+  /**
+   * userId which is currently associated to the current {@link RunContext}.
```

Comment:
Change javadoc simliar to org.eclipse.scout.rt.shared.ISession#CURRENT?
```suggestion
   * The {@code userId} which is currently associated with the current thread.  
```